.github/pull_request_template.md

```diff
@@ -0,0 +1,69 @@
+# [AAP-XXXXX] Brief Title Describing the Change
```

Comment:
should the type of change prefix (e.g.: `feat`, `fix`, `chore`) be specified in the PR title as well, or we are deprecating it in favor of `"## Type of Change"` just to be informative? does it have any impact on our release/semantic versioning processes or github actions?

Reply:
We are following that pattern but it is not enforced or stated at any place, so for now I understand is optional.

Reply:
I see. So, in this case, having `feat: [AAP-XXXXX]` or `fix: [AAP-XXXXX]` has no implications and would be optional.